flocker/node/agents/ebs.py

```diff
@@ -309,6 +311,7 @@ def _enable_boto_logging():
 _enable_boto_logging()
 
 
+@with_cmp(["requested", "discovered"])
```

Comment:
> Is it important that comparison works here (just curious, not suggesting any change)?

Yea.  One of the unit tests wants to make sure an exception with certain attribute values gets raised and the nicest spelling of that in the tests is with `assertEqual`.